src/transport.js

```diff
@@ -65,21 +65,44 @@ function generateTransport(winston, winstonTransport) {
      * @returns {void}
     */
     log(info) {
-      //  If we have specified to log in JSON format, then stringify
-      //
-      //  Also, if the `info` object which is passed from winston has more
-      //  that 2 default keys (`message` and `level`), then that means that
-      //  the `log` function was called with more two than keys
-      //  (referred to as metadata), so in that scenario we stringify the
-      //  entire message since it is not obvious what format the user wants.
+      //  This function is required since Winston passes in metadata in a strange way.
+      //  Rather than giving us a complete object containing all metadata, it hides it away
+      //  in the splat attribute and passes in only the first piece of metadata
       //  
-      //  log function, without being given a log (like below) will just log
-      //  the level, which in this case is our message.
-      if (this.json || Object.keys(info).length > 2) {
-        this.logger.log(stringify(info));
+      //  This seems to be an existing issue faced by devs:
+      //  https://stackoverflow.com/a/60866937
+      const returnMetadata = (meta) => {
+        // You can format the splat yourself
+        const splat = meta[Symbol.for('splat')];
```

Comment:
...splat?

Reply:
🤷 ask winston

explanation is in that stackoverflow ticket